project_cost_report/static/src/js/report_action.js

```diff
@@ -54,18 +54,18 @@ var ReportAction = Widget.extend(ControlPanelMixin, {
         "click .o_project_cost_report__outsourcing_total": "outsourcingTotalClicked",
         "click .o_project_cost_report__purchase_order_name": "purchaseOrderNameClicked",
     },
+    getCategoryNames(){
+        return ["product", "time", "outsourcing"];
+    },
     init(parent, action) {
         this._super.apply(this, arguments);
         this.controllerURL = action.context.url;
         this.projectId = action.context.active_id || action.params.active_id;
         this.reportContext = {
             active_id: this.projectId,
-            unfolded_categories: {
-                "product": [],
-                "time": [],
-                "outsourcing": [],
-            },
+            unfolded_categories: {},
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'unfolded_categories' is not in camel case. (camelcase)](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=4044060)